nameable.rb

```diff
@@ -0,0 +1,31 @@
+# create nameable class
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ] I think it will be better to display the full that gets raised so as to be specific in the kind of error that is expected for clarity purposes. e.g.

```
 def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
```

Reply:
Hey, @vickymarz Let me work on that.